library.sh

```diff
@@ -412,26 +412,34 @@ function start_latest_knative_serving() {
 # Install Knative Eventing in the current cluster.
 # Parameters: $1 - Knative Eventing crds manifest.
 #             $2 - Knative Eventing core manifest.
+#             $3 - Knative Eventing in memory channel manifest.
+#             $4 - Knative Eventing mt channel broker manifest.
 function start_knative_eventing() {
   header "Starting Knative Eventing"
   subheader "Installing Knative Eventing"
   echo "Installing Eventing CRDs from $1"
   kubectl apply -f "$1"
   echo "Installing Eventing core components from $2"
   kubectl apply -f "$2"
+  echo "Installing Eventing in memory channel from $3"
+  kubectl apply -f "$3"
+  echo "Installing Eventing mt channel broker from $4"
+  kubectl apply -f "$4"
```

Comment:
Are we sure we want them here? 

Is there a different user of the `start_knative_eventing` function that is expecting IMC and channel broker to be NOT there?  can we leverage those installs in separate functions?